src/platform/packages/shared/response-ops/rule_params/log_threshold/v1.test.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { omit } from 'lodash';
+import { logThresholdParamsSchema } from './v1';
+
+describe('logThresholdParamsSchema', () => {
+  const base = {
+    timeSize: 8,
+    timeUnit: 'h',
+    count: {
+      value: 1,
+      comparator: 'more than',
+    },
+    logView: {
+      logViewId: 'log-view-reference-0',
+      type: 'log-view-reference',
+    },
+    groupBy: ['geo.dest'],
+  };
+
+  const countParams = {
+    ...base,
+    criteria: [
+      {
+        field: 'bytes',
+        comparator: 'more than',
+        value: 1,
+      },
+    ],
+  };
+
+  const rationParams = {
+    ...base,
+    criteria: [
+      [
+        {
+          field: 'bytes',
+          comparator: 'more than',
+          value: 1,
+        },
+      ],
+    ],
+  };
+
+  const countParamsWithExcess = {
+    ...countParams,
+    outputIndex: '.alerts-observability.logs.alerts-default',
+  };
+
+  it('validates count params correctly', () => {
+    expect(() => logThresholdParamsSchema.validate(countParams)).not.toThrow();
+  });
+
+  it('validates ration params correctly', () => {
+    expect(() => logThresholdParamsSchema.validate(rationParams)).not.toThrow();
+  });
+
+  it('does not throw with excess fields', () => {
+    const result = logThresholdParamsSchema.validate(countParamsWithExcess);
+    expect(result).toEqual(omit(countParamsWithExcess, 'outputIndex'));
+  });
+
+  it('strips out excess fields', () => {
+    const result = logThresholdParamsSchema.validate(countParams);
+    expect(result).toEqual(omit(countParams, 'outputIndex'));
```

Comment:
`countParams` does not have `outputIndex`. Adding `countParamsWithExcess` here would make this test a duplicate, maybe we want to remove it?

Reply:
Done in https://github.com/elastic/kibana/pull/217440/commits/9c8b246db9e8a4daa683ad61667d05640a3a594a.